src/client/components/SignIn/SignIn.js

```diff
@@ -11,31 +15,54 @@ export default function SignIn({ onSubmit }) {
     onSubmit({ email, password });
   };
   return (
-    <form onSubmit={handleSubmit}>
-      <label htmlFor="email">
-        Email:{' '}
-        <input
-          type="email"
-          name="email"
-          placeholder="Write your email"
-          value={email}
-          onChange={handleEmailInput}
-          required
-        />
-      </label>
-      <label htmlFor="password">
-        Password:{' '}
-        <input
-          type="password"
-          name="password"
-          placeholder="Write your password"
-          value={password}
-          onChange={handlePasswordInput}
-          required
-        />
-      </label>
-      <button type="submit">Sign in</button>
-    </form>
+    <div>
+      <div className="signIn-icon-title">
```

Comment:
You know the drill :) 
```suggestion
      <div className="sign-in-icon-title">
```